main.py

```diff
@@ -1,116 +1,99 @@
-# Import Pyton Libraries ---------------------------------------------------
-import os
+#!/usr/bin/env python3
+from os import path, getenv, listdir
 import subprocess
+import logging
+import argparse
 
-# Logging ------------------------------------------------------------------
-q0 = input('''
-You have executed a function that will check the status of every git repository in this directory.
-Do you want to proceed (acceptable responses => 'Yes', 'yes')?  ''')
+SCRIPT_NAME = path.basename(__file__)
 
-# Set start directory
-start_dir  = input('Please provide the start directory for the program => ')
-os.chdir(start_dir)
+log_level = getattr(logging, getenv('LOGLEVEL', 'NOTSET').upper(), logging.WARNING)
+log_level = log_level if log_level else logging.WARNING
+logging.basicConfig(level=log_level)
 
-# If the user wants to proceed:
```

Comment:
Great documentation!

Well the user wasn't given a choice so I'm not sure what the point of this is.